runtime/inspector/mod.rs

```diff
@@ -681,20 +294,23 @@ impl InspectorFlags {
   }
 }
 
-struct InspectorSessions {
-  new_incoming:
-    Pin<Box<dyn Stream<Item = Box<DenoInspectorSession>> + 'static>>,
-  handshake: Option<Box<DenoInspectorSession>>,
-  established: FuturesUnordered<Box<DenoInspectorSession>>,
+/// A helper structure that helps coordinate sessions during different
+/// parts of their lifecycle.
+struct SessionContainer {
+  new_incoming: Pin<Box<dyn Stream<Item = Box<WebsocketSession>> + 'static>>,
```

Comment:
Next step is to rewrite `SessionContainer` to use a more generic "session" object, as per Discord convo `WebsocketSesssion` is not really tied to websockets and could be abstracted away as just a "proxy" that would connect to some other channels (either to use websocket transport or "in-memory")